airbyte-integrations/bases/base-standard-source-test-file/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG JDK_VERSION=17.0.1
-FROM openjdk:${JDK_VERSION}-slim
+FROM openjdk:17.0.1-slim
```

Comment:
```suggestion
FROM openjdk:${JDK_VERSION}-slim
```

Reply:
I did this change as it will be overwritten with the change in `tools/bin/publish_docker.sh`, isn't it?

Reply:
ah right good point
I think `tools/bin/build_image.sh` is the script that will affect this. `tools/bin/publish_docker.sh` should only be for platform images but same idea.

I think let's just revert the change to `tools/bin/build_image.sh` in this PR as well.  I think for this PR we just shouldn't touch any of the connector image files, as there are usually separate checks required for those changes